projects/1.20/assets/immersive-engineering/immersiveengineering/lang/zh_cn.json

```
@@ -0,0 +1,1884 @@
+  "block.immersiveengineering.slab_sheetmetal_colored_black": "黑色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_blue": "蓝色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_brown": "棕色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_cyan": "青色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_gray": "灰色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_green": "绿色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_light_blue": "淡蓝色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_light_gray": "淡灰色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_lime": "黄绿色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_magenta": "品红色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_orange": "橙色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_pink": "粉红色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_purple": "紫色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_red": "红色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_white": "白色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_colored_yellow": "黄色板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_constantan": "康铜板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_copper": "铜板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_electrum": "琥珀金板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_gold": "金板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_iron": "铁板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_lead": "铅板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_nickel": "镍板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_silver": "银板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_steel": "钢板金属台阶",
+  "block.immersiveengineering.slab_sheetmetal_uranium": "铀板金属台阶",
+  "block.immersiveengineering.slab_slag_brick": "矿渣砖台阶",
+  "block.immersiveengineering.slab_steel_scaffolding_grate_top": "格栅钢脚手架台阶",
+  "block.immersiveengineering.slab_steel_scaffolding_standard": "钢脚手架台阶",
+  "block.immersiveengineering.slab_steel_scaffolding_wooden_top": "防腐木钢脚手架台阶",
+  "block.immersiveengineering.slab_storage_aluminum": "铝台阶",
+  "block.immersiveengineering.slab_storage_constantan": "康铜台阶",
+  "block.immersiveengineering.slab_storage_copper": "铜台阶",
+  "block.immersiveengineering.slab_storage_electrum": "琥珀金台阶",
+  "block.immersiveengineering.slab_storage_lead": "铅台阶",
+  "block.immersiveengineering.slab_storage_nickel": "镍台阶",
+  "block.immersiveengineering.slab_storage_silver": "银台阶",
+  "block.immersiveengineering.slab_storage_steel": "钢台阶",
+  "block.immersiveengineering.slab_storage_uranium": "铀台阶",
+  "block.immersiveengineering.slab_treated_wood_horizontal": "防腐木台阶",
+  "block.immersiveengineering.slab_treated_wood_packaged": "防腐木台阶",
+  "block.immersiveengineering.slab_treated_wood_vertical": "防腐木台阶",
+  "block.immersiveengineering.slag_brick": "矿渣砖块",
```

```suggestion
  "block.immersiveengineering.slag_brick": "矿渣砖",
```
一般我觉得没有物品砖形态的合成出来的都不加块字。例：原版的泥砖

---

原来是这样的吗，我还以为是加s不加s呢。已改！